### ISSUE TITLE
0.1.0 Docker images failed to get pushed to GH Container Registry

### ISSUE THREAD

**Issue by arschles** [kedacore/http-add-on#197]
After the 0.1.0 release was cut, it looks like the images tagged `0.1.0` didn't get pushed.

## Expected Behavior

The release images got pushed

## Actual Behavior

They didn't

## Steps to Reproduce the Problem

  1.
  2.
  3.

## Specifications

- **KEDA Version:** *Please elaborate*
- **KEDA HTTP Add-on Version:** *Please elaborate*
- **Platform & Version:** *Please elaborate*
- **Kubernetes Version:** *Please elaborate*

**Comment by tomkerkhove** [comment]
Looks like it wasn't triggered: https://github.com/kedacore/http-add-on/actions/workflows/build_release.yml

**Comment by tomkerkhove** [comment]
I think it's due to the filter: https://github.com/kedacore/http-add-on/blob/main/.github/workflows/build_release.yml#L5

**Comment by arschles** [comment]
See this error in GH actions: https://github.com/kedacore/http-add-on/runs/2906000365?check_suite_focus=true#step:6:107

**Comment by arschles** [comment]
@tomkerkhove I re-tagged the release with `v0.1.0`, rather than the `0.1.0` that it was before, but now there is a second issue: https://github.com/kedacore/http-add-on/pull/198, which I think I fixed in that PR. When I re-run the workflow, though, it appears to be using the old YAML at the tag, rather than the new

**Comment by arschles** [comment]
Ok, it looks like this is solved. I did the following:

1. Delete the `v0.1.0` (and `0.1.0`, which I created by accident) tags
2. Remove the GH release that corresponds to the `v0.1.0` tag
3. Fix the build-release.yml GitHub actions file in #198 
4. Re-create the release
5. The operator job on the job to publish tag images (`Publish official image to GitHub Container Registry`) succeeded, the other 2 failed. I manually pushed the other two to GH Container Registry for now.

I'll open a new issue/PR to fix the release YAML for good. Closing
